test/test_xaxis.cpp

```diff
@@ -228,4 +228,18 @@ namespace xf
         EXPECT_TRUE(t2);
         EXPECT_EQ(tmp, a1);
     }
+
+    bool filter_function(const fstring& value)
```

Comment:
filter function is never used?

Reply:
indeed it should be removed. Good catch!

Reply:
Thanks :) I'll remove it